src/Data/List/Properties.agda

```diff
@@ -792,6 +799,26 @@ module _ (g : B → C) (f : A → Maybe B) where
     mapMaybe (Maybe.map g) (map f xs)  ≡⟨ mapMaybe-map _ f xs ⟩
     mapMaybe (Maybe.map g ∘ f) xs      ∎
 
+mapMaybeIsInj₁∘mapInj₁ : (xs : List A) → mapMaybe (isInj₁ {B = B}) (map inj₁ xs) ≡ xs
+mapMaybeIsInj₁∘mapInj₁ = λ where
```

Comment:
why not pattern-match ? This doesn't match the rest of the style.

Reply:
I prefer less ink on the page, especially if it the added information is completely useless/redundant (in this case repeating the name `mapMaybeIsInj₁∘mapInj₁`).

It gets even worse when you have to repeat arguments as well; maybe the case of `Any-catMaybes⁺` convinces you...

Reply:
While I greatly sympathize with this 'less ink on the page', I have had to adjust my own style when contributing to `stdlib` where I think a certain amount of uniformity of style is better.